common/src/main/java/com/hedera/block/common/constants/StringConstants.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.common.constants;
+
+import static com.hedera.block.common.constants.ErrorMessageConstants.CREATING_INSTANCES_NOT_SUPPORTED;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/** A class that hold common {@link String} literals used across projects. */
+public final class StringConstants {
+    // FILES
+    @NonNull public static final String APPLICATION_PROPERTIES = "app.properties";
+    @NonNull public static final String LOGGING_PROPERTIES = "logging.properties";
+
+    // FILE EXTENSIONS
+    @NonNull public static final String BLOCK_FILE_EXTENSION = ".blk";
+    @NonNull public static final String GZ_FILE_EXTENSION = ".gz";
+
+    // PROJECT RELATED
+    @NonNull public static final String SERVICE_NAME = "BlockStreamService";
+    @NonNull public static final String CLIENT_STREAMING_METHOD_NAME = "publishBlockStream";
+    @NonNull public static final String SERVER_STREAMING_METHOD_NAME = "subscribeBlockStream";
+    @NonNull public static final String SINGLE_BLOCK_METHOD_NAME = "singleBlock";
```

Comment:
Could we add a JavaDoc for each one of these aswell?
Do we need to decorate them with @NonNull?

Reply:
the `@NonNull` was present in the code that these were coming from, I also was wondering why it existed, thanks for pointing out, will be removed.
For the javadoc, do we really need it? Do we need to explain these literals?

Reply:
`@NonNull` removed, what about the javadoc on these literals? Do we really need comments on these literals?